fapolicy_analyzer/glade/policy_rules_admin_page.glade

```diff
@@ -1,145 +1,211 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- Generated with glade 3.38.2 -->
+<!-- Generated with glade 3.22.2 -->
 <interface>
-  <requires lib="gtk+" version="3.22"/>
-  <object class="GtkBox" id="policyRulesAdminPage">
+  <requires lib="gtk+" version="3.20"/>
```

Comment:
Is there a reason the GTK version was downgraded to 3.20?

Reply:
Nope. New commits should have this all fixed